### ISSUE TITLE
 RuntimeException php artisan VueTranslation:generate --watch=1

### ISSUE THREAD

**Issue by umairali** [tohidplus/laravel-vue-translation#1]
I updated the laravel project and run the below command after that i am receiving below error.

**Command**
`php artisan VueTranslation:generate --watch=1`

**Error**
````
 RuntimeException

  Cannot read the file "/home/test/public_html/resources/lang": permission denied (The web server user cannot read files, chmod needed)

  at vendor/elementaryframework/fire-fs/src/FireFS/FireFS.php:413
    409|         $msg = "Cannot {$action} the file \"{$path}\": permission denied";
    410|         if (!$this->isRemote($path)) {
    411|             $msg .= " (The web server user cannot {$action} files, chmod needed)";
    412|         }
  > 413|         return new \RuntimeException($msg);
    414|     }
    415|
    416|     /**
    417|      * Write in the file

      +21 vendor frames
  22  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

   Whoops\Exception\ErrorException

  PHP Startup: Unable to load dynamic library 'imagick.so' (tried: /opt/cpanel/ea-php73/root/usr/lib64/php/modules/imagick.so (libMagickWand.so.5: cannot open shared object file: No such file or directory), /opt/cpanel/ea-php73/root/usr/lib64/php/modules/imagick.so.so (/opt/cpanel/ea-php73/root/usr/lib64/php/modules/imagick.so.so: cannot open shared object file: No such file or directory))

  at Unknown:0
    1|

      +1 vendor frames
  2   [internal]:0
      Whoops\Run::handleShutdown()

**Comment by tohidplus** [comment]
It seems that the permission of the lang directory has been changed for some reason. Give it read permission and try again.

**Comment by mikaelprag** [comment]
This is more likely due to an update of the "fire-fs" package. Downgrading it to version 1.0.1 solved this issue for me. A compose update fetched the new 1.1.0 version and broke laravel-vue-translation.

**Comment by tohidplus** [comment]
Please use the new patched version `2.0.1` of the package and downgrade the FireFs to `1.0.0`
